presentation-api/index.html

```diff
@@ -0,0 +1,118 @@
+---
+feature_name: Presentation API
+chrome_version: 51
+feature_id: 6676265876586496
+---
+
+<h3>Background</h3>
+
+<p>
+  This sample illustrates the use of
+  <a href="https://w3c.github.io/presentation-api/">Presentation API</a>,
+  which gives the ability to access external presentation-type displays and use
+  them for presenting web content. The <code>PresentationRequest</code> object
+  is associated with a request to initiate or reconnect to a presentation made
+  by a controlling browsing context and it takes in a presentation request URL
+  when constructed. A presentation can be started by calling the
+  <code>start()</code> method on the <code>PresentationRequest</code> object.
+</p>
+<p>
+  Presentation API is supported only on desktop for now. Media Router has to be
+  enabled for the demo to work: <code>chrome://flags/#media-router</code>.
+</p>
+
+{% capture initial_output_content %}
+<p>
+  <button id="createRequest">new PresentationRequest()</button>
+</p>
+<p>
+  <button id="start" disabled>presentationRequest.start()</button>
+</p>
+<p>
+  <input id="presentationId" type="text" placeholder="presentationId">
+  <button id="reconnect" disabled>presentationRequest.reconnect(presentationId)</button>
+</p>
+<p>
+  <button id="close" disabled>presentationRequest.close()</button>
+  <button id="terminate" disabled>presentationRequest.terminate()</button>
+</p>
+{% endcapture %}
+{% include output_helper.html initial_output_content=initial_output_content %}
+
+{% capture js %}
+// App ID EF1A139F is registered in the Google Cast SDK Developer
+// Console and points to the following custom receiver:
+// https://googlechrome.github.io/samples/presentation-api/receiver/index.html
+var presentationUrl = 'http://google.com/cast/#__castAppId__=EF1A139F';
+var presentationRequest;
+var presentationConnection;
+
+function updatePresentationAvailability(displaysAvailable) {
+  ChromeSamples.log('Presentation displays are ' +
+      (displaysAvailable ? 'available' : 'unavailable') + '.');
+}
+
+function setConnection(connection) {
+  presentationConnection = connection;
+  document.querySelector('#close').disabled = false;
+  document.querySelector('#terminate').disabled = false;
+  presentationConnection.onclose = function() {
+    ChromeSamples.log('Connection closed.');
+  };
+  presentationConnection.onterminate = function() {
+    ChromeSamples.log('Connection terminated.');
+  };
+}
+
+document.querySelector('#createRequest').addEventListener('click', function() {
```

Comment:
You may want to move this function above `setConnection` as `document.querySelector('#createRequest')` is the first button user is supposed to click.